Tensile/BenchmarkProblems.py

```diff
@@ -43,6 +43,41 @@
 from .SolutionWriter import SolutionWriter
 from .TensileCreateLibrary import writeSolutionsAndKernels, writeCMake, buildObjectFileNames
 
+
+def generateForkedSolutions (problemType, forkedParameters, commonSolutionParameters):
+  
+  solutions = []
+  numForked = len(forkedParameters)
+
+  ############################################################################
+  # this creates a set or solutions based on the forked parameters using
+  # a set of common parameters from which to fork from
```

Comment:
These kinds of header comments should be in docstrings at the beginning of functions so they are more accessible.